liquidity/autoloop_testcontext_test.go

```diff
@@ -344,6 +344,21 @@ func (c *autoloopTestCtx) autoloop(step *autoloopStep) {
 		return c.manager.numActiveStickyLoops() == 0
 	}, defaultEventuallyTimeout, defaultEventuallyInterval, "failed to"+
 		" wait for sticky loop counter")
+
+	// Since we're checking if any false-positive swaps were dispatched we
+	// need to give some time to autoloop to possibly dispatch them.
+	time.Sleep(500 * time.Millisecond)
```

Comment:
nit: perhaps it's better to do a `<-time.Afer(500 * time.Millisecond)` inside the `select`.